x/foundation/keeper/params.go

```diff
@@ -39,12 +41,18 @@ func (k Keeper) UpdateParams(ctx sdk.Context, params foundation.Params) error {
 	}
 
 	// clean up relevant authorizations
+	// sort it for the determinism (line/lbm-sdk#943)
+	urlRemovedSorted := make([]string, 0, len(urlRemoved))
 	for url, removed := range urlRemoved {
-		url := url
-
 		if !removed {
 			continue
 		}
+		urlRemovedSorted = append(urlRemovedSorted, url)
+	}
+	sort.Strings(urlRemovedSorted)
```

Comment:
Is this ordering same with the current finschia-1's genesis list?

Reply:
It's not necessary. It would be sufficient to ensure all nodes have the urls in the same order.